server/src/main/java/org/apache/iotdb/db/doublelive/OperationSyncLogService.java

```diff
@@ -0,0 +1,214 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iotdb.db.doublelive;
+
+import org.apache.iotdb.db.conf.IoTDBDescriptor;
+import org.apache.iotdb.db.engine.fileSystem.SystemFileFactory;
+import org.apache.iotdb.db.writelog.io.LogWriter;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
+
+public class OperationSyncLogService implements Runnable {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(OperationSyncLogService.class);
+
+  private static final String logFileDir =
+      IoTDBDescriptor.getInstance().getConfig().getOperationSyncLogDir();
+  private static final long logFileValidity =
+      IoTDBDescriptor.getInstance().getConfig().getOperationSyncLogValidity() * 1000L;
```

Comment:
Why we multiply 1000 here?  This will cause the default value of logFileValidity become 30000 and we will sleep 30000 seconds later. Please double check here.

Reply:
I multiply 1000 in OperationSyncLogSerive because I use System.currentTimeMillis().